README.md

```diff
@@ -5,17 +5,44 @@ Bogus documentation page.
 
 ## Requirements (per assignment)
 To make a **documentation page** featuring:
-- 
+- A < main> element with id="main-doc" holding the main content
+- Within main-doc, at least 5 < section> elements with class="main-section"
+- Each of them with a < header> element with the section's topic
+- Also with an individual id="Name_of_section" (underscores for spaces)
+- The .main-section elements should have at least 10 < p> elements total
+- They should also have at least 5 < code> elements total
+- Also at least 5 < li> elements total (not each)
+- A < nav> element with id="navbar"
+- Inside navbar, a < header> element with the topic at the beginning
+- Also < a> elements with class="nav-link" to each .main-section element, with text corresponding to the < header> of each section, linked to it
+- .navbar should be on the left and always visible
+- There has to be at least one media query
+
 
 ## How I'm planning it to look, and why
-Main reference: example from [FreeCodeCamp's CodePen](https://codepen.io/freeCodeCamp/full/NdrKKL) passing all tests from this exercise.
+Main reference: example from [FreeCodeCamp's CodePen](https://codepen.io/freeCodeCamp/full/NdrKKL) passing all tests from this exercise. But... let's make it FUN (and innecesarily painful), shall we? Since coding is the closest there is to arcane magic (you write some obscure words in the right order and stuff happens) let's turn this documentation page into a spellbook explaining 5 different spells. We'll need appropiate fonts, colors... maybe even hover animations! We'll be here *FOREVER*.
+
+The structure will have to be the same (menu on the left, content on the right, sections...) but we will style the text and box elements differently than the example. This looks like a chance to try the CSS grid too.
 
 ## Steps
```

Comment:
These steps are great as technical steps, but what I would like is to see functional steps. Taking this project into mind, steps would be something like:

1. Draw page drafts and decide on the visual elements of the page (colour scheme, fonts, etc.) and content.
2. Implement page layout with placeholders for sidebar and main and make sure they resize as expected on different screen sizes.
  a. Whatever technical steps this implies. At the beginning it's helpful to define the technical steps, but afterwards it won't be necessary. The focus is not how we achieve something but what we achieve.
3. User needs to be able to navigate to the main section/introduction by clicking the side link
  a. As technical steps then it will be to create the sidebar, link to the section, style both things.
4. User wants to see the remaining content for the page (or just section 2, or whatever you consider an adequate chunk of work). 
  a. Reply the work on the introduction but blah blah...